### PR TITLE
Slightly tweaked a README.md

### PR DESCRIPTION
**Pull request by xobotyi**
Just a bit of small improvements for readme page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,8 +191,6 @@ export default class SubmitButton extends Component {
 
 `Array.isArray`: see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) for details about unsupported older browsers (e.g. <= IE8) and a simple polyfill.
 
-`Object.keys`: see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) for details about unsupported older browsers (e.g. <= IE8) and a simple polyfill. This is only used in `dedupe.js`.
```
Comment (newyork-anthonyng) [inline README.md]: Good catch on this 👍

File: README.md [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
+[![Supported by Thinkmill](https://thinkmill.github.io/badge/heart.svg)](https://thinkmill.com.au/?utm_source=github&utm_medium=badge&utm_campaign=classnames)
 
 A simple JavaScript utility for conditionally joining classNames together.
 
 Install with [npm](https://www.npmjs.com/), [Bower](https://bower.io/), or [Yarn](https://yarnpkg.com/):
 
-npm:
-```sh
+```bash
+# via NPM
```
Comment (newyork-anthonyng) [inline README.md]: Should we preserve the casing of `npm`, `Bower`, and `Yarn`?

Comment (xobotyi) [reply README.md]: Yikes written as I usually do, but it differs.
Shold be npm, Bower and yarn.
I'll fix it a bit later.

Comment (xobotyi) [reply README.md]: Fixed.

**Comment by JedWatson** [comment]
Thanks @xobotyi!

**Pull request merged**